<launch>
    <node name="record_odom" pkg="ros_basics_project" type="calc_distance_traveled_action_server.py" output="screen">

    </node>
</launch>